modcc/backends/base.hpp

```diff
@@ -8,15 +8,18 @@
 #include <stdexcept>
 #include <string>
 
-#include "options.hpp"
 #include "token.hpp"
 #include "textbuffer.hpp"
-#include "util/meta.hpp"
 
-namespace arb {
+enum class simdKind {
+    none, avx2, avx512
+};
+
 namespace modcc {
 
-using arb::util::enable_if_t;
+template <bool V, typename R = void>
+using enable_if_t = typename std::enable_if<V, R>::type;
+
```

Comment:
I suppose that's related to the separation of the utility lib from the arbor source, right?

Reply:
Exactly, yes. I plan to put it back once we have a separate utility source directory :)